server/TelegramServerConfig.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef TELEGRAMSERVER_CONFIG_HPP
+#define TELEGRAMSERVER_CONFIG_HPP
+
+#include <QString>
+#include "DcConfiguration.hpp"
+
+namespace Telegram {
+
+namespace Server {
+
+class Config {
+public:
+    explicit Config(const QString &fileName = QString());
+
+public:
```

Comment:
Please follow the Qt coding style and omit extra `public` sections.

I would also move const getters right after the constructor to follow the common convention to have 'slots' (though there are no slots)  such as `load()` and `save()` after getters.

The common order is
```
public:
    // constructor
    // getter methods
public slots:
    // setters and mutable "action" methods
signals:
```

Reply:
This is not QObject at all

Reply:
I didn't say anything about QObject and I took 'slots' into a quote and said that "there are no slots" obviously because the class is not a QObject.

See e.g. https://code.woboq.org/qt5/qtbase/src/gui/text/qfont.h.html for references.

"Actions" (please don't mix up with QAction or QQuickAction) usually go after getters and setters.